aktivgraph/README.md

```diff
@@ -0,0 +1,12 @@
+# /ag/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for Aktivgraph (ag) resources.
+
+## Contact
+This space is administered by:  
+
+**Wolfgang Kremser**  
+*Researcher*  
+[Salzburg Research](https://www.salzburgresearch.at/)  
+Austria
+<wolfgang.kremser@salzburgresearch.at>
+GitHub: [kremserwo](https://github.com/kremserwo)
```

Comment:
Note that several lines do not retain their line breaks. You need to end these lines with either two space chars (`  `) or an HTML break (`<br>`).